pkg/pipeline/terraformed.go

```diff
@@ -49,30 +48,35 @@ type TerraformedGenerator struct {
 }
 
 // Generate writes generated Terraformed interface functions
-func (tg *TerraformedGenerator) Generate(cfg *config.Resource, parametersTypeName string) error {
+func (tg *TerraformedGenerator) Generate(cfgs map[*config.Resource]string, apiVersion string) error {
```

Comment:
nitpick: if the only reason to use a map is to couple type name with the configuration, you can consider coming up with a struct type to transit the information and then send an array of that object. Something like the following:
```go
type TerraformedInput struct { // maybe private?
  config.Resource
  ParametersTypeName string
}
```

Reply:
Done!